setup/src/main.ts

```diff
@@ -0,0 +1,59 @@
+import * as core from '@actions/core';
+import * as exec from '@actions/exec';
+
+
+/**
+    Setup sprkl function
+ */
+(async () => {
+    // get the input values from the action
+    const sprklVersion = core.getInput('version');
+    const analyze = core.getInput('analyze');
+    const setEnv = core.getInput('setenv');
+
+    // run sprkl install command
+    const installCmd = `npx @sprkl/scripts@${sprklVersion} install`;
+    await exec.exec(installCmd);
```

Comment:
In some cases we know the installation may fail due to problems linking in /usr/local/bin ...
I think we should at least add $HOME/.sprkl/bin to the $PATH env variable